### ISSUE TITLE
Large loss and nan loss

### ISSUE THREAD

**Issue by FeiDao7943** [facebookresearch/votenet#164]
**** EPOCH 002 ****
Current learning rate: 0.001000
Current BN decay momentum: 0.500000
2023-03-27 11:14:37.992102
 ---- batch: 010 ----
mean box_loss: nan
mean center_loss: nan
mean heading_cls_loss: nan
mean heading_reg_loss: nan
mean loss: nan
mean neg_ratio: 0.254590
mean obj_acc: 0.254375
mean objectness_loss: nan
mean pos_ratio: 0.009082
mean sem_cls_loss: nan
mean size_cls_loss: nan
mean size_reg_loss: nan
mean vote_loss: nan
 ---- batch: 020 ----
mean box_loss: nan
mean center_loss: nan
mean heading_cls_loss: nan
mean heading_reg_loss: nan
mean loss: nan
mean neg_ratio: 0.000000
mean obj_acc: 0.000000
mean objectness_loss: nan
mean pos_ratio: 0.000000
mean sem_cls_loss: nan
mean size_cls_loss: nan
mean size_reg_loss: nan
mean vote_loss: nan

/////////////

 ---- batch: 080 ----
mean box_loss: 36476394887.889046
mean center_loss: 36476394887.495895
mean heading_cls_loss: 0.000000
mean heading_reg_loss: 0.012006
mean loss: 364819971942.011230
mean neg_ratio: 0.742920
mean obj_acc: 0.906894
mean objectness_loss: 0.160109
mean pos_ratio: 0.053662
mean sem_cls_loss: 2.860292
mean size_cls_loss: 2.870165
mean size_reg_loss: 0.136136
mean vote_loss: 5600754.862075
 ---- batch: 090 ----
mean box_loss: 0.804259
mean center_loss: 0.334334
mean heading_cls_loss: 0.000000
mean heading_reg_loss: 0.020176
mean loss: 25.172875
mean neg_ratio: 0.758154
mean obj_acc: 0.868903
mean objectness_loss: 0.155064
mean pos_ratio: 0.048584
mean sem_cls_loss: 2.875596
mean size_cls_loss: 2.889619
mean size_reg_loss: 0.160787
mean vote_loss: 1.347937
 ---- batch: 100 ----
mean box_loss: 3856308962287565668352.000000
mean center_loss: 3856308962287565668352.000000
mean heading_cls_loss: 0.000000
mean heading_reg_loss: 0.029361
mean loss: 38563091424330962173952.000000
mean neg_ratio: 0.808154
mean obj_acc: 0.857417
mean objectness_loss: 0.147534
mean pos_ratio: 0.039111
mean sem_cls_loss: 2.843654
mean size_cls_loss: 2.867544
mean size_reg_loss: 0.160195
mean vote_loss: 953150979688.176392
 ---- batch: 110 ----
mean box_loss: 0.724312
mean center_loss: 0.253483
mean heading_cls_loss: 0.000000
mean heading_reg_loss: 0.024551
mean loss: 22.057832
mean neg_ratio: 0.727441
mean obj_acc: 0.861654
mean objectness_loss: 0.159908
mean pos_ratio: 0.055566
mean sem_cls_loss: 2.833293
mean size_cls_loss: 2.864183
mean size_reg_loss: 0.159860
mean vote_loss: 1.118188
 ---- batch: 120 ----
mean box_loss: 19054326959700664.000000
mean center_loss: 19054326959700664.000000
mean heading_cls_loss: 0.000000
mean heading_reg_loss: 0.029966
mean loss: 190544650858704576.000000
mean neg_ratio: 0.786963
mean obj_acc: 0.841404
mean objectness_loss: 0.150460
mean pos_ratio: 0.043848
mean sem_cls_loss: 2.877827
mean size_cls_loss: 2.862448
mean size_reg_loss: 0.168548
mean vote_loss: 136601700196.556839

**Comment by FeiDao7943** [comment]
I think it may caused by the pointnet2 installed successfully but not correctly. 
Envirnment: win11, Pytorch v1.1, TensorFlow v1.14, CUDA 10.0, cuDNN v7.4. Nvidia 4060 Laptop GPU
